.github/workflows/ansible-lint.yaml

```diff
@@ -1,4 +1,4 @@
-name: Ansible Lint # feel free to pick your own name
```

Comment:
Is there any reason for this space?

Reply:
Convention. Some linters would comply otherwise.